fdbserver/TagPartitionedLogSystem.actor.cpp

```diff
@@ -2136,92 +2131,92 @@ void populateBitset(boost::dynamic_bitset<>& bs, std::vector<uint16_t>& ids) {
 // All prior versions to the chosen RV must also be recoverable.
 // TODO: unit tests to stress UNICAST
 Version getRecoverVersionUnicast(std::vector<Reference<LogSet>>& logServers,
-                                 std::vector<std::tuple<int, std::vector<TLogLockResult>>>& logGroupResults,
-                                 Version minDVEnd,
-                                 Version minKCVEnd) {
-	std::vector<std::vector<uint16_t>> tLogLocIds;
+                                 std::tuple<int, std::vector<TLogLockResult>>& logGroupResults,
```

Comment:
can be `const`?

Reply:
Done. (And, turned a couple of other arguments into const).